filters/node.js

```diff
@@ -0,0 +1,29 @@
+import flatten from 'flat'; 
+
+export class NodeFilter {
+
+    async parseResource( resource ) {
+        const segments = [];
+        for(const [key, value] of Object.entries(flatten(resource))){
+            if (key.indexOf("@")===-1) {
+                const seg = {
+                    sid: key,
+                    str: value,
+                };
+                if (key.endsWith("_one")||key.endsWith("_other")) {
```

Comment:
this assumes the source has only 2 forms. we should consider any of the 6 forms to be pluralized.

Reply:
Added.